### PR TITLE
improvement(migrate): help output

### PR DESCRIPTION
**Pull request by Jolg42**
Screen Shot 2020-11-26 at 11 22 23

### CODE REVIEW

**Comment by albertoperdomo** [comment]
Suggestions:

dev -> Create migrations from Prisma schema, apply migrations to database, generate artifacts (Prisma Client).
reset -> Reset your database and apply all migrations.

deploy -> Apply migrations to update the database schema.
resolve -> Resolve issues with database migrations, i.e. baselining databases, failed migrations, hotfixes.

I know some of these now feel too long? If I had an easy way to preview how ti looks in terminal I would have...
What do you think?

**Comment by albertoperdomo** [comment]
BTW, I like the structuring of these into development and not development...

File: src/packages/migrate/src/commands/MigrateCommand.ts [hunk 1]
```diff
@@ -31,12 +31,15 @@ ${chalk.bold('Usage')}
...
-      deploy   Deploy your migrations to your database
-       reset   Reset your database, all data will be lost
-     resolve   Resolve your database migration state
+         dev   Easily create, apply migrations, generate artifacts (Prisma Client) 
+       reset   Reset your database and apply all migrations, all data will be lost
+
+${chalk.bold('Commands for staging/production')}
+
+      deploy   Deploy your migrations
+     resolve   Resolve your database migration which failed
```
Comment (Jolg42) [inline src/packages/migrate/src/commands/MigrateCommand.ts:42]:
```suggestion
         dev   Create migrations from your Prisma schema, apply them to the database, generate artifacts (Prisma Client)
       reset   Reset your database and apply all migrations

${chalk.bold('Commands for staging/production')}

      deploy   Apply migrations to update the database schema
     resolve   Resolve issues with database migrations (baseline, failed migration, hotfix).
```

File: src/packages/migrate/src/commands/MigrateReset.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ export class MigrateReset implements Command {
   }
 
   private static help = format(`
-Reset your database and reapply migrations
+Reset your database and apply all migrations, all data will be lost
```
Comment (Jolg42) [inline src/packages/migrate/src/commands/MigrateReset.ts:28]:
```suggestion
Reset your database and apply all migrations
```

File: src/packages/migrate/src/commands/MigrateDeploy.ts [hunk 3]
```diff
@@ -23,7 +23,7 @@ export class MigrateDeploy implements Command {
   }
 
   private static help = format(`
-Deploy your migrations to your database.
+Deploy your migrations to your staging/production database
```
Comment (Jolg42) [inline src/packages/migrate/src/commands/MigrateDeploy.ts:26]:
```suggestion
Apply migrations to update the database schema in staging/production
```

File: src/packages/migrate/src/commands/MigrateResolve.ts [hunk 4]
```diff
@@ -26,7 +26,7 @@ export class MigrateResolve implements Command {
   }
 
   private static help = format(`
-Resolve your database migration state
+Resolve your database migration which failed in your staging/production database
```
Comment (Jolg42) [inline src/packages/migrate/src/commands/MigrateResolve.ts:29]:
```suggestion
Resolve issues with database migrations (baseline, failed migration, hotfix) in staging/production
```

File: src/packages/migrate/src/commands/MigrateDev.ts [hunk 5]
```diff
@@ -39,7 +39,7 @@ export class MigrateDev implements Command {
 
   private static help = format(`${
     process.platform === 'win32' ? '' : chalk.bold('🏋️  ')
-  }Migrate your database with confidence
+  }Easily create, apply migrations, generate artifacts (Prisma Client)
```
Comment (Jolg42) [inline src/packages/migrate/src/commands/MigrateDev.ts:42]:
```suggestion
  }Create migrations from your Prisma schema, apply them to the database, generate artifacts (Prisma Client)
```

**Comment by Jolg42** [comment]
will be in 2.13.0-dev.16

**Pull request merged**